Makefile

```diff
@@ -53,12 +62,16 @@ deploy: deploy-controller deploy-cleanup-agent
 
 # Deploy controller in the configured Kubernetes cluster in ~/.kube/config
 deploy-controller: manifests
-	cd config/manager && kustomize edit set image controller=${CONTROLLER_IMG}
+	cd config/manager && \
+		kustomize edit set image controller=${CONTROLLER_IMG} && \
+		kustomize edit set label pool:${SYSTEM_POOL}
```

Comment:
@paulosjca Would you rather depend on a specific pool since it is an environment variable in the deployment? Or would you rather just have me set it to go to the nodes with a "default-system-pool" label set to "true"?

Reply:
The concept of "default pool" is that it is something used by the controller to assign a (client, driver or server) pod to a pool when there is none specified. The actual pod always has a "pool" label set as a node selector.

In the case of the controller itself, there is no entity that would look at the pod configuration and set the  "pool" label, so it has to be set in the deployment step. The code as you have it is correct.